test/performance/requirements.txt

```diff
@@ -0,0 +1,4 @@
+kubernetes==30.1.0
+PyYAML==5.4.1
+PyYAML==6.0.2
```

Comment:
Two different versions for the same package looks wrong.

Reply:
Fixed!